show/main.py

```diff
@@ -1384,7 +1384,12 @@ def runningconfiguration():
 def all(verbose):
     """Show full running configuration"""
     cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    output = json.loads(run_command(cmd, display_cmd=verbose, return_cmd=True))
```

Comment:
Should we check the output of run_command?

Reply:
You mean the format of output or bgpraw in output?

Reply:
geoffrey@vm-04:~$ python3
Python 3.6.9 (default, Nov 25 2022, 14:10:45) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> json.loads("")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
>>>

Reply:
Maybe that should be checked in sonic-cfggen?
sonic-cfggen guarantees that `--print-data` could be json.dump.